.docker/createQuery.sql

```diff
@@ -0,0 +1,141 @@
+CREATE TABLE ROOM_TYPE ( -- Typ pokoju, do typów mozna potem dodac jakies kolory/zdjecia

```

Comment:
komentarza po polsku są fuj XD
może jakaś tablica, może cokolwiek z notatkami by się przydało, jakas stronę postawić niby confluence z takimi sprawami? jakiekolwiek krótkie opisy byłby spoko(poza kodem)

Reply:
Te komentarze były tylko dla Ciebie żebyś sobie przeczytał co gdzie jest. Sam mówiłeś że docelowo nie chcemy tworzyć bazy z SQL. Bez sensu jest pisać komentarze po angielsku jak to jest tylko dla nas, tym bardziej że ten plik będzie usunięty w następnych commitach.
Bawienie się w jakieś confluence jest imo useless chyba, że ktoś inny by chciał się w to bawić, bo ja nie widzę celu.

Reply:
Już go nima